qchem/tests/test_convert.py

```diff
@@ -18,14 +18,14 @@
 import sys
 
 import pytest
-
-pytest.importorskip("openfermion")
+from openfermion import QubitOperator
 
 import pennylane as qml
-from openfermion import QubitOperator
 from pennylane import numpy as np
 from pennylane import qchem
 
+pytest.importorskip("openfermion")
```

Comment:
Does the order matter for the lines 
`from openfermion import QubitOperator` and `pytest.importorskip("openfermion")`?

Reply:
Modified it to `openfermion = pytest.importorskip("openfermion")`.